native/effektio/src/api.rs

```diff
@@ -85,7 +85,22 @@ mod api {
     }
 }
 
-fn init_logging(filter: Option<String>) -> Result<()> {
-    platform::init_logging(filter)?;
-    Ok(())
+fn init_logging(log_dir: String, filter: Option<String>) -> Result<()> {
+    platform::init_logging(log_dir, filter)
+}
+
+async fn report_bug(
```

Comment:
clippy is complaining about this one, too

Reply:
Fixed